src/helpers/formatters/file.js

```diff
@@ -1,17 +1,17 @@
 import fs from 'fs'
+import * as musicMetadata from 'music-metadata'
```

Comment:
@Borewit Can it be `import metadata from 'music-metadata'`?

Reply:
I can add default exports: Borewit/music-metadata#978